packages/ndla-ui/src/Notion/Notion.tsx

```diff
@@ -1,120 +1,249 @@
+      width: 100% !important;
+      padding: 0;
+    }
+  }
+`;
+const TextWrapper = styled.div`
+  width: 75%;
+  ${mq.range({ until: breakpoints.tabletWide })} {
+    width: 100%;
+  }
+  font-family: ${fonts.serif};
+  ${fonts.sizes('18px', '28px')};
+  ${ContentWrapper} .c-figure.expanded + & {
+    width: 100%;
+  }
 `;
 
-const AuthorsContainer = styled.div`
-  display: flex;
-  flex-wrap: wrap;
-  margin: 0 0 1.5rem;
-  font-size: 0.875rem;
+const ImageElement = styled.img``;
 
-  p {
-    margin: 0;
+const fadeInMediaKeyframe = keyframes`
+   0% {
+     opacity: 0;
+     height: auto;
+   }
+   100% {
+     opacity: 1;
+   }
+ `;
+const fadeOutMediaKeyframe = keyframes`
+   0% {
+     opacity: 1;
+     height: auto;
+   }
+   100% {
+     opacity: 0;
+     height:0;
+     overflow: hidden;
+   }
+ `;
 
-    &:not(:only-child) {
-      margin-right: 1rem;
-    }
+const ImageWrapper = styled.div`
+  float: right;
+  width: 25%;
+  padding-left: ${spacing.normal};
+  position: relative;
+  ${mq.range({ until: breakpoints.tabletWide })} {
+    width: 100%;
+    padding-left: 0;
   }
 `;
 
-const LabelsContainer = styled.div`
-  display: flex;
+const ExpandVisualElementButton = styled(Button)`
+  position: absolute;
+  right: 8px;
+  bottom: 8px;
+  transition: all ${animations.durations.normal};
+  &,
+  &:focus,
+  &:active {
+    background-color: rgba(255, 255, 255, 0.65);
+  }
+  color: ${colors.brand.primary};
+  border-radius: 50%;
+  border: 0;
+  width: 40px;
+  height: 40px;
+  display: inline-flex;
+  justify-content: center;
   align-items: center;
-  font-size: 0.875rem;
-  margin-bottom: 1rem;
+  svg {
+    transition: transform ${animations.durations.normal} ease-out;
+  }
+  ${ContentWrapper}:hover & {
+    background-color: #fff;
+    svg {
+      transform: scale(1.2);
+    }
+  }
+`;
 
-  p {
-    margin: 0;
+const ExpandIcon = styled.span`
+  ${ExpandVisualElementButton}.expanded & {
+    display: none;
+  }
+`;
+const CollapseIcon = styled.span`
+  display: none;
+  ${ExpandVisualElementButton}.expanded & {
+    display: inline-block;
   }
+`;
+
+const ClearWrapper = styled.div`
+  clear: both;
+`;
 
-  > :not(:last-child) {
-    margin-right: 0.5rem;
+const MediaContainer = styled.div`
+  opacity: 0;
+  height: 0;
+  overflow: hidden;
+  &.expanded {
+    animation-name: ${fadeInMediaKeyframe};
+    animation-duration: 2.8s;
+    opacity: 1;
+    height: auto;
+  }
+  &.fadeOut {
+    animation-name: ${fadeOutMediaKeyframe};
+    animation-duration: 2.8s;
   }
 `;
 
-const Label = styled.span`
+const LabelsContainer = styled.div`
+  display: flex;
+  align-items: center;
   ${fonts.sizes('14px', '24px')};
-  font-weight: 600;
-  padding: 0 0.25rem;
-  background-color: ${colors.brand.greyLightest};
-  vertical-align: center;
+  font-family: ${fonts.sans};
+  margin: ${spacing.small} 0;
 `;
 
+type VisualElementProps = {
+  type: 'video' | 'other';
+  element: React.ReactNode;
```

Comment:
Legg heller inn ReactNode i importene så slepper vi å ha `React.` her.
```suggestion
  element: ReactNode;
```